### DISCUSSION TITLE
What is pyarmor group?

### DISCUSSION THREAD

**Discussion by MasIgor** [dashingsoft/pyarmor#1195] [Q&A] [answered]
Hi all!

First off, i have already checked the faq and read the licensing part of the documentation. but there are still things that are not clear for me:

1) beside that it requires no internet, what is the difference with pyarmor pro?
2) where can pyarmor group be bought? in the link https://order.mycommerce.com/product?vendorid=200089125&productid=301044051 there is only basic, pro and upgrade. but no group...
3) for the PRO version, is the internet requrired for every obfuscation or only to register the license?

thank you very much

Igor

**Comment by jondy** [comment, marked as answer]
for the PRO version, internet is requrired for every obfuscation
pyarmor group will be available in 2 weeks with pyarmor 8.2.x

also check this
https://pyarmor.readthedocs.io/en/latest/how-to/register.html#using-group-license

**Reply by MasIgor** [reply]
thank you Jondy, I have already read all this. But i still dont have clear what the difference between pro and group is. Only the requirement to have internet? The pro can be installed on multiple devices using the generated file. looks like the group version too.. both unlock all features, both are just for one product. what is then the real advantage/benefit/difference between the two?

**Reply by jondy** [reply]
Yes, the only difference is no internet required for group. In future the group may be more features.
